website/versioned_docs/version-27.0/CodeTransformation.md

```diff
@@ -110,6 +110,12 @@ interface TransformOptions<OptionType> {
   // names are copied from babel: https://babeljs.io/docs/en/options#caller
   supportsDynamicImport: boolean;
   supportsExportNamespaceFrom: boolean;
+  /**
+   * The value is:
+   * - `false` if Jest runs without Node ESM flag `--experimental-vm-modules`
+   * - `true` if the file extension is defined in [extensionsToTreatAsEsm](https://jestjs.io/docs/configuration#extensionstotreatasesm-arraystring)
```

Comment:
this should ideally link relatively (this is 27.0 docs, it shouldn't link to `latest`)

Reply:
Done